spec/rubocop/cop/rspec/change_by_zero_spec.rb

```diff
@@ -365,5 +365,12 @@
         expect { foo }.to change { Foo.bar }.by(1)
       end
     RUBY
+
+    expect_no_offenses(<<~RUBY)
+      it do
+        subject; change(Foo, :bar).by(0)
```

Comment:
I would very much rather this cop to raise an exception than to swallow such a spec.

Reply:
Indeed, that example might be unnecessary. However, it makes sense that code like `change(foo, :bar).by(0)` doesn't produce an error, so perhaps we could simply remove it from the test examples.